test/data/javascript/tests.js

```diff
@@ -3443,6 +3443,11 @@ exports.test_data = {
             '}'
           ]
         },
+        {
+          comment: "Issue #1683 - switch-case wrong indentation",
+          input: 'switch (x) { case 0: if (y == z) { a(); } else { b(); } case 1: c(); }',
+          output: 'switch (x) {\n    case 0:\n        if (y == z) {\n            a();\n        } else {\n            b();\n        }\n    case 1:\n        c();\n}'
```

Comment:
Please change these to multi-line format (the array of strings).

Reply:
Sure, I will make the changes